pkg/command/backup/cmd.go

```diff
@@ -163,7 +165,11 @@ func (cmd *command) run(args []string) error {
 		ok = true
 	}
 	if cmd.Flag("skip-schema").Changed {
-		props["skip_schema"] = cmd.purgeOnly
+		props["skip_schema"] = cmd.skipSchema
```

Comment:
I think it was not possible to change the skip_schema to ture, as purgeOnly is true only if someone just to explicitly clean expired snapshots.
Nice finding.